src/screens/profile/styles.ts

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components/native';
+
+export const HeadingContainer = styled.Text`
+  margin-top: 86px;
+  margin-bottom: 16px;
+  word-break: break-all;
+  width: 100%;
+`;
+export const ButtonContainer = styled.TouchableOpacity`
+  margin-top: 30%;
+  margin-left: 30px;
+`;
+export const ButtonPasswordContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
+export const ButtonEmailContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
+export const ButtonPhoneContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
+export const EmailText = styled.Text`
+  font-family: 'Manrope';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  margin-top: -6.3%;
+  color: #000000;
+`;
+
+export const ButtonBlank = styled.TouchableOpacity`
+  box-sizing: border-box;
+  height: 0px;
+  width: 110%;
+  border: 0.5px;
+`;
+export const ButtonBottomContainer = styled.TouchableOpacity`
+  margin-top: 3%;
+  margin-left: 0%;
+  width: 100%;
+`;
+
+export const MagentaButtonContainer = styled.View`
+  flex-direction: top;
```

Comment:
I don't think this is standard practice? While it may work on IOS, it crashes the app on Android - you should be using `row` or `column`.